xml/depl_maintenance.xml

```diff
@@ -1503,6 +1503,295 @@ migrate</screen>
+	 <para>
+	  <filename>/var/log/nova/nova-compute</filename> on the source and
+	  target &compnode;s.
+	 </para>
+	</listitem>
+	<listitem>
+	 <para>
+	  <command>/var/log/nova/*.log</command> (especially log files for the
+	  conductor, scheduler and placement services) on the &contrnode;s.
+	 </para>
+	</listitem>
+       </itemizedlist>
+       <para>
+	It can happen that active instances and instances with heavy
+	loads cannot be live migrated in a reasonable time. In that case,  you
+	can abort a running live-migration using the <command>nova
+	live-migration-about <replaceable>MIGRATION-ID</replaceable></command>
+	command. You can then perform the upgrade of the specific node at a
+	later time.
+       </para>
+       <para>
+	Alternatively, it is possible to force the completion of
+	the live migration by using the <command>nova
+	live-migration-force-complete
+	<replaceable>MIGRATION-ID</replaceable></command> command. However,
+	this might pause the instances for a prolonged period of time and have
+	a negative impact on the workload running inside the instance.
+       </para>
+      </answer>
+     </qandaentry>
+     <qandaentry>
+      <question>
+       <para>
+	Node has failed during OS upgrade.
+       </para>
+      </question>
+      <answer>
+       <para>
+	Possible reasons include an incorrect repository setup or package
+	conflicts. Check the <filename>node-upgrade.log</filename> log file on the
+	node. Check the repositories on node using the <command>zypper
+	lr</command> command. Make sure the required repositories are
+	available. To test the setup, install a package manually or run the
+	<command>zypper dup</command> command (this command is executed by the
+	upgrade script). Fix the repository setup and run the failed upgrade
+	step again. If custom package versions or version locks are in place,
+	make sure that they don't interfere with the <command>zypper dup</command> command.
+       </para>
+      </answer>
+     </qandaentry>
+     <qandaentry>
+      <question>
+       <para>
+	Node does not come up after reboot.
+       </para>
+      </question>
+      <answer>
+       <para>
+	It could happen that it takes the node too long to come up, which cause
+	a timeout. Check the node manually, and make sure it is online. Repeat then the step.
+       </para>
+      </answer>
+     </qandaentry>
+     <qandaentry>
+      <question>
+       <para>
+	N number of nodes were provided to compute upgrade using
+	<command>crowbarctl upgrade nodes node_1,node_2,...,node_N</command>,
+	but less then N were actually upgraded.
+       </para>
+      </question>
+      <answer>
+       <para>
+	If the live migration cannot be performed for certain nodes due to a timeout,
+	&crow; upgrades only the nodes that it was able to
+	live-evacuate. Since some nodes have been upgraded, it is possible that
+	more resources will available for live-migration if you try to run this
+	step again. See also question on live migration failure.
```

Comment:
BTW, is this block understandable? I understand it because I know what is it trying to describe (and because I wrote it), but I fear it's not written well.

Reply:
I understand it, and I'm not even an expert. :-)

Reply:
Sorry folks, I don't understand it. What does "...upgrades only the nodes that it was able to live-evacuate in the specified time." mean?

Reply:
"See also question on live migration failure." -> "See the question on live migration failure for more information" (and then link to that question if possible)

Reply:
> Sorry folks, I don't understand it. What does "...upgrades only the nodes that it was able to live-evacuate in the specified time." mean?

It means that it can only upgrade the nodes that can be evacuated before hitting the specified timeout. Is that correct @jsuchome ?

Reply:
When more nodes are provided for parallel upgrade (in the way of `crowbarctl upgrade nodes node1,node2,node3`) crowbar first tries to live-evacuate each node one by one (so _not_ in parallel). After that it means they do not have any VM's running so it can upgrade all of them. 

live-evacuation of a node is live-migration of each VM from such node

Reply:
> It means that it can only upgrade the nodes that can be evacuated before hitting the specified timeout. Is that 
> correct @jsuchome ?

Right. The main point of this issue is that it could be a subset of the nodes that user wanted.